Podfile

```diff
@@ -0,0 +1,18 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '6.0'
```

Comment:
So we can remove it? If it's not doing anything I'd prefer that, just adds confusion otherwise.